package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-subsection-generator",
-  "version": "8.0.2",
+  "version": "8.0.3",
```

Comment:
Given that this requires major changes in 2 **peer** libs, I'm thinking maybe we should consider this a breaking change? What do you think?

Reply:
I think it makes sense. By applying the other suggestion (dropping support for `react-redux@7`), we now definitely have a breaking change, as users might be forced to update their code after upgrading this library.

Updating internal dependencies wouldn't require a major version bump, but that's not the case here.